docs/docs/guides/getting_started.md

```diff
@@ -62,6 +62,7 @@ detectors:
 
 cameras:
   camera_1: # <------ Name the camera
+    enabled: True # <----- Enable/disable camera
```

Comment:
I'm not sure if it makes sense to put it here, this section is the minimal config and this isn't necessary